.azure-pipelines/ultimate-pipeline.yml

```
@@ -10,9 +10,14 @@ trigger:
   paths:
     exclude:
       - docs/*
```

Wildcards aren't supported!
```suggestion
      - docs/
```